src/view/treeNodes/fileChangeNode.ts

```diff
@@ -253,8 +254,8 @@ export class FileChangeNode extends TreeNode implements vscode.TreeItem {
 		return this;
 	}
 
-	openFileCommand(): vscode.Command {
-		return openFileCommand(this.filePath);
+	openFileCommand(opts: Pick<vscode.TextDocumentShowOptions, 'preview'>): vscode.Command {
```

Comment:
When we're opening all of the files, we likely don't want to open them as a preview. In order to support this, callers can now specify whether or not files should open as a preview.